Fixed of speaker screen design

NUmeroAndDev opened the pull request:
## Issue
- None

## Overview (Required)
- Fixed of speaker screen design.
  - Fixed divider margin.
  - Fixed text style.
      

## Link
- https://github.com/DroidKaigi/conference-app-2019/pull/488#issuecomment-455023753


## Screenshot
Before | After
:--: | :--:
<img src="https://user-images.githubusercontent.com/13705006/51168328-45912780-18ec-11e9-93ec-aa5c5bd74d7f.png" width="300" /> | <img src="https://user-images.githubusercontent.com/13705006/51369529-4a95e700-1b37-11e9-9d52-7de8f3edb218.png" width="300" />

takahirom commented:
Although the policy of the font to use is not decided, Lekton does not include Japanese. Do you think that it is a good thing to replace them all? 🤔

NUmeroAndDev commented:
@takahirom
Is there a possibility that the font may change if Lekton's text also includes Japanese in the gallery?

Sorry, I thought Lekton's text would never change.

It is okay for you to close this pull request.

takahirom commented:
I will give priority to this matter to designers today

takahirom commented:
Sorry, I can not see Designer 😭 
Can you paste before after image when Japanese speaker? 🙏

takahirom commented:
日本語ですみません！
本当にありがとうございます🙇
デザイナーにチャット上で今の状況を伝える必要があって、
そのためにEmojiAppCompatTextViewでかつTextAppearance.App.HogeHogeを使っているスクリーンショットも必要になるので、頂けますか？その上で多分決められると思います🙏
いろいろ貢献いただいていてありがとうございます🙇

NUmeroAndDev commented:
@takahirom 
> EmojiAppCompatTextViewでかつTextAppearance.App.HogeHogeを使っているスクリーンショット

notosans のフォントを適用した場合で合っていますでしょうか?

<img src="https://user-images.githubusercontent.com/13705006/51253538-eb728e00-19e1-11e9-9030-1572a018e560.png" width="45%">

takahirom commented:
それだとspeaker_descriptionのtextAppearanceにTextAppearance.App.Body1があたっていない気がしていていますがあたってますかね。。？
時間取れないようでしたら自分で用意します🙇

NUmeroAndDev commented:
@takahirom 
>speaker_descriptionのtextAppearanceにTextAppearance.App.Body1があたっていない気がしていていますがあたってますかね。。？

すいません、適用されていませんでした 🙇 
<img src="https://user-images.githubusercontent.com/13705006/51286979-c5c4a380-1a38-11e9-9337-a6b053fc720d.png" width="45%" >

takahirom commented:
The policy has been decided. I use Lakton fonts only for always English and numbers display, others want to use Noto. Can you make such a change?
方針決まりました。常に英語や数字の表示のところだけLaktonで、ほかはNotoでいきたいです。そのように変更お願いできますか？ 🙇

NUmeroAndDev commented:
@takahirom 
2 つほど質問したいのですが、

>常に英語や数字の表示のところだけLaktonで、ほかはNotoでいきたいです。

* 例えばここの Roboto が指定されている箇所も Noto ということでしょうか?
<img width="1198" alt="2019-01-17 13 43 00" src="https://user-images.githubusercontent.com/13705006/51296116-e3f0ca80-1a5d-11e9-943d-95a7db3be741.png">


* Gallery のデータて更新していただけるのでしょうか?

takahirom commented:
このルールで統一で、そこもNotoということだそうです！🙏

NUmeroAndDev commented:
@takahirom 
I fixed with [this policy](https://github.com/DroidKaigi/conference-app-2019/pull/488#issuecomment-455023753).

Fonts may have changed on other screens as well.
Because I modified `TextAppearance.App.HogeHoge`. 

Please check again 🙇

takahirom commented:
Thank you for tacking design issue!! 🎨
